.github/workflows/nextjs-integration-test.yml

```diff
@@ -193,3 +182,29 @@ jobs:
           path: |
             nextjs-test-results.json
             test-summary.md
+
+  post_to_slack:
```

Comment:
I think we need to move this to on-nextjs-release-publish.yml, since we don't plan to spam channel per each PR's test results for now.